run.go

```diff
@@ -67,20 +70,55 @@ func run(context *cli.Context) error {
 		return errors.New("invalid runtime config")
 	}
 
-	if err := create(context.Args().First(),
+	consolePath := context.String("console")
+
+	var wg sync.WaitGroup
+	var console Console
+	var err error
+	var consoleState *term.State
+
+	// if consolePath is /dev/ptmx or /dev/pts/ptmx
+	// means that we have to allocate a new pts
+	if consolePath == ptmxPath || consolePath == ptsPtmxPath {
```

Comment:
Are you sure this works in case you use `$(tty)` in your docker command ? If I use `$(tty)`, I always end up with something like /dev/pts/11 where 11 can be any number. How is that supposed to work for this case ?